capstone-server/src/main/java/com/hcl/capstoneserver/config/GlobalErrorHandler.java

```diff
@@ -85,5 +90,32 @@
                 HttpStatus.UNAUTHORIZED
         );
     }
+
+    @ExceptionHandler(EmailAlreadyExistsException.class)
+    public final ResponseEntity<Object> handleEmailExistsException(EmailAlreadyExistsException ex) {
+        ArrayList<Map<String, String>> errors = errorsMap("email", ex.getMessage());
+        return new ResponseEntity<>(
+                new DefaultValidationErrorResponse(HttpStatus.BAD_REQUEST, "User Already Exists", errors),
```

Comment:
change the message to "An account with this email address already exists. please sign in or use another email"